docs/source/api.rst

```diff
@@ -184,6 +308,174 @@ HVAC
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       PSC    
+``systemType``                     string                                               yes       
+``conditionedFloorAreaServed``     float    ft2                         >0              no        PSC
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+
+
+HVAC Control Systems
+~~~~~~~~~~~~~~~~~~~~
+
+If any HVAC systems are specified, a single control system can be entered in ``...building.systems.hvac.hvacControlSystems``.
+
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default             Notes
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       HVACControl1
+``heatingSeason``                  object                                                                             Thermostat settings for heating season
+``coolingSeason``                  object                                                                             Thermostat settings for cooling season
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+
+``heatingSeason`` and ``coolingSeason`` objects share the following schema:
+
+=================================  =======  ==========================  ==============  ========  ========================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default                   Notes
+=================================  =======  ==========================  ==============  ========  ========================  ==============================================       
+``setpointTemp``                   float    F                                           no        Heating: 67, Cooling: 78  
+``setbackTemp``                    float    F                                           no        null                      Also referred to as setup temperature for the cooling season.
+``setbackStartHour``               integer  time                        0-23            no        null    
+``totalWeeklySetbackHours``        integer  hrs/week                    0-168           no        null    
```

Comment:
I've seen you have `null` defaults for these properties. What do you think about enforcing a real value if the property is provided? If there's no available information or the user doesn't want to provide a value, he or we can just omit the property.

If there is a special meaning on using `null`, then we can keep it around as a default.

Reply:
I'm not sure I understand your meaning. Let me clarify my intention though - I made the default values of these properties `null` to indicate that our assumptions do not include setbacks. If this object is not defined, we will assume a static `setpointTemp`.